app/Compile.hs

```diff
@@ -95,10 +95,10 @@ convGlobal g =
     GFunction (Function n u t cs) -> GFunction (Function n u (baseToReturn t) (map funClauseReturn cs))
     GAbstract (Abstract n u t) -> GAbstract (Abstract n u (baseToReturn t))
 
-funClauseReturn (FunClause patts term) = FunClause (map pattEval patts) (baseToReturn term)
+funClauseReturn (FunClause tel patts term rhs catchall unreachable) = FunClause undefined (map pattEval patts) (baseToReturn term) undefined undefined undefined -- TODO
 
 funClauseEval :: IR.FunClause' IR.NoExt Param.PrimTy Param.RawPrimVal -> IR.FunClause' IR.NoExt Param.PrimTy (TypedPrim Param.PrimTy Param.RawPrimVal)
-funClauseEval (FunClause patts term) = FunClause (map pattEval patts) (baseToReturn term)
+funClauseEval (FunClause tel patts term rhs catchall unreachable) = FunClause undefined (map pattEval patts) (baseToReturn term) undefined undefined undefined --TODO
```

Comment:
Is it possible to fill in the `undefined`s yet, or is more work elsewhere needed first?

Reply:
Unfortunately (much!) more work elsewhere is needed first.